src/main/java/oscar/task/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package oscar.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline task that contains description of task and deadline.
+ */
+public class Deadline extends Task {
+    private final LocalDateTime deadline;
+
+    /**
+     * Public constructor for deadline.
+     *
+     * @param description Details of deadline.
+     * @param deadline Deadline of task with date and time information.
+     */
+    public Deadline(String description, LocalDateTime deadline) {
+        super(description, "D");
+        this.deadline = deadline;
+    }
+
+    /**
+     * Obtains string representation of deadline.
+     *
+     * @return Information of deadline.
+     */
+    @Override
+    public String toString() {
```

Comment:
Nice to see that line length < 120 characters standard has been applied here! 👍 😊